pages/contests/index.js

```diff
@@ -0,0 +1,65 @@
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../../styles/Home.module.css";
```

Comment:
Let's add the style to _app.js or to document (see nextsjs docs)

Reply:
done